src/deploy.ts

```diff
@@ -88,7 +88,11 @@ type DeployTargetInfo =
 /** Deploy a project to ObservableHQ */
 export async function deploy(deployOptions: DeployOptions, effects = defaultEffects): Promise<void> {
   Telemetry.record({event: "deploy", step: "start", force: deployOptions.force});
-  effects.clack.intro(`${inverse(" observable deploy ")} ${faint(`v${process.env.npm_package_version}`)}`);
+  effects.clack.intro(
+    `${inverse(" observable deploy ")}${
+      process.env.npm_pacakge_version ? faint(` v${process.env.npm_package_version}`) : ""
```

Comment:
This gets baked into the code during build, so it shouldn’t be necessary to check this.

https://github.com/observablehq/framework/blob/2bba99bfe0ff58a56790230b6bd61f0b5682de3c/build.js#L17